bundle.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM scratch
+
+# Core bundle labels.
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=nvidia-network-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=v1.1.0
+LABEL operators.operatorframework.io.bundle.channel.default.v1=v1.1.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.7.1+git
```

Comment:
searching commits, i see we migrated to operator-sdk 1.8.0

https://github.com/Mellanox/network-operator/commit/a9c8812bc65d6a2cf93ad345d183e6fc9b3e17ea

does this have any meaning here ?

Reply:
This is just a label, this file is auto-generated by operator-sdk

Reply:
@rollandf  which version of operator-sdk did you use?

Reply:
```
$ operator-sdk version
operator-sdk version: "v1.18.0", commit: "c9c61b6921b29d731e64cd3cc33d268215fb3b25", kubernetes version: "1.21", go version: "go1.17.7", GOOS: "linux", GOARCH: "amd64"
```